config/initializers/rollbar.rb

```diff
@@ -7,7 +7,9 @@
   # Disable Rollbar in "test" and "development" environments
   config.enabled = false if Rails.env.test? || Rails.env.development?
 
-  config.anonymize_user_ip = true
```

Comment:
Does `anonymize_user_ip` not do enough in this case? I was under the impression that is removed some enough octals for it to be considered anonymized? Scrubbing headers probably makes sense though.

Reply:
> Does `anonymize_user_ip` not do enough in this case? I was under the impression that is removed some enough octals for it to be considered anonymized? Scrubbing headers probably makes sense though.

Yeah, I didn't really understand what was happening, according to this:

https://docs.rollbar.com/docs/ruby#section-managing-sensitive-data

`config.anonymize_user_ip = true` should ' semi-anonymization on the captured IP address by masking out the least significant bits' but that was not what I was seeing in rollbar.

The headers still had what look like the 'real' IP address and the 'masked' address is completely different?! You can take a look in Rollbar if you like – I'd love another opinion to understand what was happening.

Reply:
@pezholio how wedded to keeping the `anonymize_ip` are you, I love to get this in to stop the potenrial dob leak, as I don't fully understand what Rollbar is doing, my prefered option is to not send the IP at all, but I would be happy either way.

Reply:
Yeah, let's leave it as you've done here. I don't think the IP is useful in Rollbar anyway